tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java

```diff
@@ -67,7 +72,7 @@ public void testFailureResponsePayload() throws Exception {
 
         // response size
         JsonArray checks = json.getJsonArray("checks");
-        Assert.assertEquals(checks.size(), 2, "Expected two check responses");
+        Assert.assertEquals(checks.size(), 4, "Expected four check responses");
```

Comment:
Can you the 4 procedures listed in the json array in the following for loop?

Reply:
Initial test wasn't doing that but it's a good idea. I suggest that propose this as an enhancement in RC1

Reply:
The following for loop does check the 2 procedures though. 
```
for (JsonValue check : checks) {
            String id = asJsonObject(check).getString("name");
            switch (id) {
                case "successful-check":
                    verifySuccessPayload(check);
                    break;
                case "failed-check":
                    verifyFailurePayload(check);
                    break;
                default:
                    throw new IllegalArgumentException("Unexpected response payload structure");
            }
        }
```

Reply:
What I suggested is to add two more cases.

Reply:
Sorry @Emily-Jiang. I don't get it : the loop is not limited to 2 iteration but does all the array so should go through the 4 checks. I'm probably missing something...

Reply:
@Emily-Jiang @antoinesd I think I can see where misunderstanding can be: Emily is proposing to add two more cases to check all 4 procedures but that is already done in the loop because all successful procedures (SuccessfulHealth, SuccessfulLiveness and SuccessfulReadiness) have the same name "successful-check" so all four procedures are verified. Sorry if I missed something.